pkg/commands/deploy.go

```diff
@@ -158,11 +161,12 @@ func NewDeployTopologyUpdaterCommand(commonOpts *CommonOptions, opts *deployOpti
 			if opts.clusterPlatform == platform.Unknown {
 				return fmt.Errorf("cannot autodetect the platform, and no platform given")
 			}
-			return rte.Deploy(la, rte.Options{
-				Platform:         opts.clusterPlatform,
-				WaitCompletion:   opts.waitCompletion,
-				RTEConfigData:    commonOpts.RTEConfigData,
-				PullIfNotPresent: commonOpts.PullIfNotPresent,
+			return updater.Deploy(la, updater.Options{
```

Comment:
another option could be to have different functions (and/or subpackages) taking the same options instead of passing through `UpdaterType`. So we can have, say, `updater/rte` and `updater/nfd`